benchmark/CMakeLists.txt

```
@@ -15,8 +15,8 @@ if (CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)
     set(XTENSOR_INCLUDE_DIR ${xtensor_INCLUDE_DIRS})
 endif ()
 
-message(STATUS "Forcing tests build type to Release")
-set(CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build." FORCE)
+#message(STATUS "Forcing tests build type to Release")
+#set(CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build." FORCE)
```

```suggestion
message(STATUS "Forcing tests build type to Release")
set(CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build." FORCE)
```
I presume this was just for your convince. Similar below